AutomatticTracks/build.gradle

```diff
@@ -10,10 +10,8 @@ repositories {
 }
 
 dependencies {
-    def sentryVersion = "5.0.1"
```

Comment:
I'm wondering if/how BoM works across modules. We sometimes define a version in the project build file and use it in multiple modules. But I guess we could still define bom in the project level build.gradle (feels a bit weird). Anyway, it's fine for this use case since it fixes dependabot, but I'd discuss it with the team before we start using BoM everywhere.